lib/api/WP_Auth0_Api_Change_Email.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Contains WP_Auth0_Api_Change_Email.
+ *
+ * @package WP-Auth0
+ *
+ * @since 3.9.0
+ */
+
+/**
+ * Class WP_Auth0_Api_Change_Email to update a user's email at Auth0.
+ */
+class WP_Auth0_Api_Change_Email extends WP_Auth0_Api_Abstract {
+
+	/**
+	 * Default value to return on failure.
+	 */
+	const RETURN_ON_FAILURE = false;
```

Comment:
Since failure doesn't always return `false` (sometimes `0` or something else), then it's nice to have it in one place. These also API classes extend an abstract which can look for this constant and use it in a parent method.